x-pack/plugins/dataset_quality/server/utils/queries.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { QueryDslQueryContainer } from '@elastic/elasticsearch/lib/api/types';
+
+function isUndefinedOrNull(value: any): value is undefined | null {
```

Comment:
This is also defined in the observability plugin.

Reply:
Addressed in [db8ce22](https://github.com/elastic/kibana/pull/172462/commits/db8ce22dada1cb1a820a8405c4a1db6da02be72c).